awaitwhat/blocker.py

```diff
@@ -1,11 +1,12 @@
 import inspect
+import random
 
 
 def blockers(task):
     """What does this task wait for?"""
     w = task._fut_waiter
     if not w:
-        return
+        return [f'<Not blocked {random.random()}>']
```

Comment:
@dimaqq Should we merge https://github.com/dimaqq/awaitwhat/pull/15 first?

Reply:
Maybe it's easier to merge the name change later...